bref

```diff
@@ -248,4 +248,60 @@ $app->command('deployment stack-name [--region=]', function (string $stackName,
     return 0;
 })->descriptions('Displays the latest deployment logs from CloudFormation. Only the logs from the last 24 hours are displayed. Use these logs to debug why a deployment failed.');
 
+$app->command('dashboard [--port=]', function (string $port='8000', SymfonyStyle $io) {
+    $exeFinder = new ExecutableFinder();
+    if (! $exeFinder->find('docker')) {
+        $io->error(
+            'The `docker` command is not installed.' . PHP_EOL .
+            'Please follow the instructions at https://docs.docker.com/install/'
+        );
+
+        return 1;
+    }
+
+    if (! $exeFinder->find('serverless')) {
+        $io->error(
+            'The `serverless` command is not installed.' . PHP_EOL .
+            'Please follow the instructions at https://bref.sh/docs/installation.html'
+        );
+
+        return 1;
+    }
+
+    $servelessInfo = new Process(['serverless', 'info']);
+    $servelessInfo->run();
+
+    if (!$servelessInfo->isSuccessful()) {
+        $io->error('The command `serverless info` failed' . PHP_EOL . $servelessInfo->getErrorOutput());
+    }
+
+    $servelessInfoOutput = $servelessInfo->getOutput();
+
+    $region = [];
+    preg_match('/region: ([a-z0-9-]*)/', $servelessInfoOutput, $region);
+    $region = $region[1];
+
+    $stack = [];
+    preg_match('/stack: ([a-zA-Z0-9-]*)/', $servelessInfoOutput, $stack);
+    $stack = $stack[1];
+
+    $io->writeln("Region : $region, Stack : $stack");
+
+    $process = new Process(['docker', 'run', '--rm', '-p', $port.':8000', '-v', getenv('HOME').'/.aws:/root/.aws:ro', '--env', 'STACKNAME='.$stack, '--env', 'REGION='.$region, 'bref/dashboard']);
```

Comment:
In the future (https://github.com/docker/cli/pull/1498#issuecomment-505974526) there will be a `docker run --pull` command.

Right now we could do:

- `docker pull`
- `docker run`

to always have the latest version running.